Bench project appears to be broken

When i follow the instructions in the readme i get an error saying 


```bash
$ java -Xprof -jar target/scala-2.12/bitcoin-s-bench-assembly-0.0.4-SNAPSHOT.jar > output.txt
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.NoClassDefFoundError: scala/Function0
        at java.lang.Class.getDeclaredMethods0(Native Method)
        at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
        at java.lang.Class.privateGetMethodRecursive(Class.java:3048)
        at java.lang.Class.getMethod0(Class.java:3018)
        at java.lang.Class.getMethod(Class.java:1784)
        at sun.launcher.LauncherHelper.validateMainClass(LauncherHelper.java:544)
        at sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:526)
Caused by: java.lang.ClassNotFoundException: scala.Function0
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 7 more
```

--- a/bench/README.md
+++ b/bench/README.md
@@ -8,10 +8,30 @@ You can assembly the jar like this
 
 ```
 sbt bench/assembly
+[info] Updating secp256k1jni...
+[info] Done updating.
+[info] Updating core...
+[info] Done updating.
+[warn] There may be incompatibilities among your library dependencies.
+[warn] Run 'evicted' to see detailed eviction warnings
+[info] Updating ...
+[info] Done updating.
+[warn] There may be incompatibilities among your library dependencies.
+[warn] Run 'evicted' to see detailed eviction warnings
+[info] Compiling 4 Scala sources to /home/chris/dev/bitcoin-s-core/core/target/scala-2.12/classes ...
+[info] Done compiling.
+[info] Compiling 56 Scala sources to /home/chris/dev/bitcoin-s-core/core/target/scala-2.12/classes ...
+[info] Done compiling.
+[info] Strategy 'discard' was applied to 11 files (Run the task at debug level to see details)
+[info] Packaging /home/chris/dev/bitcoin-s-core/bench/target/scala-2.12/bitcoin-s-bench-assembly-f85fcf-1550326975891-SNAPSHOT.jar ...
+[info] Done packaging.
 ```
+
 
 and then run with
 
 ```
-$ java -Xprof -jar bench/target/scala-2.11/bench-assembly-0.0.1-SNAPSHOT.jar > output.txt
+$ java -Xprof -jar /home/chris/dev/bitcoin-s-core/bench/target/scala-2.12/bitcoin-s-bench-assembly-f85fcf-1550326975891-SNAPSHOT.jar > output.txt
 ```
+
+The `hash` and `timestamp` are in the console output above. In this example it is `f85fcf` and `1550326975891`

--- a/build.sbt
+++ b/build.sbt
@@ -162,9 +162,9 @@ lazy val rpc = project
 lazy val bench = project
   .in(file("bench"))
   .enablePlugins()
+  .settings(commonSettings: _*)
   .settings(assemblyOption in assembly := (assemblyOption in assembly).value
     .copy(includeScala = true))
-  .settings(commonSettings: _*)
   .settings(
     libraryDependencies ++= Deps.bench,
     name := "bitcoin-s-bench",

